### PR TITLE
Add some tests

### PR DESCRIPTION
**Pull request by DmytroOrlov**

### CODE REVIEW

File: server/test/controllers/MockMessageApi.scala [hunk 1]
```diff
@@ -0,0 +1,34 @@
+package controllers
+
+import play.api.i18n.{Lang, Messages, MessagesApi}
+import play.api.mvc
+import play.api.mvc.Result
+import play.mvc.Http.RequestHeader
+
+class MockMessageApi extends MessagesApi {
```
Comment (novakov-alexey) [inline server/test/controllers/MockMessageApi.scala]: Can scalamock lib do this automatically?

Comment (DmytroOrlov) [inline server/test/controllers/MockMessageApi.scala]: probably

Comment (novakov-alexey) [inline server/test/controllers/MockMessageApi.scala]: Better than native scala mock lib?

Comment (DmytroOrlov) [inline server/test/controllers/MockMessageApi.scala]: what?

Comment (novakov-alexey) [inline server/test/controllers/MockMessageApi.scala]: Mockito is better than native scala mock library?

Comment (DmytroOrlov) [inline server/test/controllers/MockMessageApi.scala]: it depends, I recommend to use scalamock first, otherwise than refactor, and finally use mockito

Comment (novakov-alexey) [inline server/test/controllers/MockMessageApi.scala]: Ok

File: server/test/controllers/TemplateTest.scala [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package controllers
+
+import org.scalatestplus.play.PlaySpec
+import play.api.i18n.Lang
+import play.api.mvc.{Flash, Session}
+import play.api.test.Helpers._
+
+class TemplateTest extends PlaySpec {
+/*
```
Comment (novakov-alexey) [inline server/test/controllers/TemplateTest.scala]: do we need this commented code in master?

**Pull request merged**